### DISCUSSION TITLE
How to create a mix between a DataTable and a TreeView

### DISCUSSION THREAD

**Discussion by emlavey** [carbon-design-system/carbon-components-svelte#1856] [Q&A] [answered]
Good morning,
I just discovered your Components. I use a software ("Bomist") and I want to make the same view which is a mix between a TreeView and a DataTable for another personal project. Is it possible to do this directly by options in TreeView or by options in DataTable Component ?
![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/56973855/cef29fca-9642-4eee-816c-7da6868956e6)

The main thing is to be able to group by several objects. Then be able to expand the category to see all objects in a Data table format
I tried to make a DataTable with expandable rows. Then put an sub DataTable in the expanded part. But the header of the sub-table cannot be hidden and the columns do not line up to his parent. Do you have any ideas for making this table ?

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/56973855/2f294420-3028-4727-b434-42c8235a1cc1)

Thank you for all the help or ideas you will bring me.

**Comment by theetrain** [comment, marked as answer]
There appears to be a feature request in Svelte Core, but I suspect it'll take a while to land on a core solution: https://github.com/carbon-design-system/carbon/issues/12617

For the interim, your best bet may be to use CSS overrides in your nested DataTable.
